src/output.rs

```diff
@@ -420,10 +420,10 @@ pub fn newfmt_envelope(
     if crate::OUTPUT_FORMAT != 0 {
         meta.push_u(METADATA_OBJECT_FORMAT, crate::OUTPUT_FORMAT)
     }
-    if token != "" {
+    if !token.is_empty() {
         meta.push_s(METADATA_OBJECT_TOKEN, token)
     }
-    if attrs.len() > 0 {
+    if !attrs.is_empty() {
```

Comment:
Same remark here.

Reply:
Would you prefer me to un-stage these kind of changes from this PR?

Reply:
Probably not - I recognize a lost cause when I see one, I hope.  I guess I needed to vent about it - IMO it's an annoying amount of (non-ergonomic) drama caused by somebody introducing a second way to do something that was already possible to do.  Maybe there's a justification for this in the realm of generics.  Anyway, let's just do it the way you've done it and be done with it.

Reply:
Generally, I find clippy useful and it helped me to improve code generally but we don't need to be clippy "certified" from my perspective. I see how "if something is not empty" reads clunky and possibly does not help the understanding.

Reply:
I undid all the changes that introduced .is_empty() conditionals.